### ISSUE TITLE
Is there a way to use CodeMirror's `placeholder` extension?

### ISSUE THREAD

**Issue by davidcelis** [voracious/ink-mde#53]
Hello! First off, thank you so much for Ink. I've been wanting to plug a markdown editor into my personal Rails site for a while and was dreading it, but I couldn't believe how easy it was to put Ink into an import map and just plug it in.

I was wondering if there's an out-of-the-box way to use CodeMirror 6's [placeholder extension](https://github.com/codemirror/view/blob/main/src/placeholder.ts) to get a… well, placeholder into my editor 😄 It's easy enough to provide an initial `doc` but it would be even better to get true placeholder functionality so that it isn't considered content and is automatically overridden as soon as content is typed in.

**Comment by voracious** [comment]
Hey there, @davidcelis! It's definitely possible to supply a CodeMirror extension to Ink, but I actually think it makes sense to support this one natively! If you don't want to wait, you can supply an extension to Ink like this, but keep in mind the API for extensions might change in future versions (as I'm still figuring out the best interface for different extension types).

```ts
import { placeholder } from '@codemirror/view'
import { ink } from 'ink-mde'

ink(document.querySelector('#editor'), {
  extensions: [
    { type: 'default', value: placeholder() },
  ],
})
```

**Comment by davidcelis** [comment]
Oh, awesome! Native support would definitely be best and I'm happy to wait for that 🙌

**Comment by voracious** [comment]
@davidcelis this is available in [`v0.20.1`](https://github.com/voracious/ink-mde/releases/tag/v0.20.1).
